conformance/lang/expressions/is-expr/is_expr.balt

```diff
@@ -1841,11 +1841,11 @@ function init() {
     io:println(d is int:Unsigned32); // @output true
     io:println(d is byte); // @output true
 
-    any e = table key(x) [{x: 1, y: [10.2, "A"]}];
-    io:println(e is TableType); // @output true
+    any e = table [{x: 1, y: [10.2, "A"]}];
+    io:println(e is TableType); // @output false
     io:println(e is table<map<float|string>>); // @output false
-    io:println(e is table<Record> key<int>); // @output true
-    io:println(e is anydata); // @output true
+    io:println(e is table<Record> key<int>); // @output false
+    io:println(e is anydata); // @output false
```

Comment:
Shall we add a true case as well since all is-checks are false?

Reply:
Added via [fa5b4dc](https://github.com/ballerina-platform/ballerina-spec/pull/1106/commits/fa5b4dc823b0b1ed0be3d6e5cd1a20a2e9e41f50)